trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/import_airfoil_geometry.py

```diff
@@ -11,34 +11,29 @@
 #           May 2021, R. Erhard
 #           Jun 2021, E. Botero
 #           Aug 2021, M. Clarke
-#           Aug 2022, R. Erhard
 
 # ----------------------------------------------------------------------
 #  Imports
-# ----------------------------------------------------------------------
+# ---------------------------------------------------------------------- 
 from SUAVE.Core import Data  
-import scipy.interpolate as interp
 import numpy as np
-import os
+from scipy import interpolate
 
 ## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil
-def  import_airfoil_geometry(airfoil_geometry_files, npoints = 200,surface_interpolation = 'cubic'):
-    """This imports an airfoil geometry from a text file  and stores
+def import_airfoil_geometry(airfoil_geometry_file, npoints = 200,surface_interpolation = 'cubic'):
+    """This imports an airfoil geometry from a text file  and store
     the coordinates of upper and lower surfaces as well as the mean
     camberline
     
     Assumptions:
     Works for Selig and Lednicer airfoil formats. Automatically detects which format based off first line of data. Assumes it is one of those two.
```

Comment:
does this still work with both lednicer and selig files?

Reply:
yes... this is actually the old script (pre Racheal's edits) since we discovered that there are some bugs. All I did was add functions to increase the density and the leading and trailing edge